docs/sources/reference/commandline/cli.md

```diff
@@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+### Daemon user namespace support
```

Comment:
I completely disagree.

Documentation should never be dumbed down or details removed especially simply because there's other avenues. Have you ever sat on IRC and answered the same question day in and day out because the docs didn't say something? 

I think detail is > readability in any sense. If you don't n understand something, learn it. If you don't need to read something, skim/skip it but there's no excuse to leave ANYTHING undocumented or subvert detail.

Now, if the documentation is a tutorial for a specific task, maybe, but, if it applies, it should be included, even if under a separate header for easy skipping or under a tl/dr.

I don't disagree, that overall, the documentation should be easy to read however, honestly, this is a technical subject and aiming for grade 8 readability (assuming Flesch-Kin aid formula) seems a little low. Perhaps, a target, but every part falling under this would seriously deprive documentation of important detail.

Ps. IMHO & I only read the last message, out of context: I apologize if this doesn't make sense because of that.


-------- Original message --------
From: moxiegirl <notifications@github.com> 
Date: 03-11-2015  11:33 AM  (GMT-07:00) 
To: docker/docker <docker@noreply.github.com> 
Subject: Re: [docker] Proposal: User namespace support for re-mapped root per
  daemon setting (#11253) 

In docs/sources/reference/commandline/cli.md:

> @@ -349,6 +350,31 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
>  not where the primary development of new functionality is taking place.
>  Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
>  
> +### Daemon user namespace support
The question "why can't I set this per container" is easy to understand. If we provide an answer, it should be equally easy to understand.

For example, here the first three sentences provides a pretty good explanation of filesystem layering. These sentences have a Grade Level 16 of readability. Most of our documentation is written at 8 or lower. Effectively, most of our readers would find this very elegantly written explanation hard to parse.

Moreover, in designing the information in this section, do I really want to distract the reader from their actual focus task only to provide three long sentences to explain part of what is meant by this single phrase Due to the requirement that the filesystem layers need to have user/group ownership modification to make this remapping useful,?

If we need to say anything, I'd rather write "Due to file system limitations, you cannot map on a per container basis" and move on. Yes, I agree with you, there are readers at the contributor level who may want the implementationdetail but we have other avenues such as IRC or forums for those questions.

—
Reply to this email directly or view it on GitHub.